CHANGES.md

```diff
@@ -83,7 +83,8 @@
 
 ## Known Issues
 
-* Fixed X (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
+* (Python) Performance impact due to occasional erroneous conversions from bytes/str to
```

Comment:
This sounds like a regression that should be addressed on the release branch.

Reply:
Looks like this is getting a fix soon after all (#12654), so we might just be able to cherry pick the fix in.